src/main/java/com/github/sylvainlaurent/maven/swaggervalidator/semantic/validator/path/MimeTypeValidator.java

```diff
@@ -0,0 +1,80 @@
+package com.github.sylvainlaurent.maven.swaggervalidator.semantic.validator.path;
+
+import static org.apache.commons.collections4.ListUtils.emptyIfNull;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import com.github.sylvainlaurent.maven.swaggervalidator.semantic.node.path.OperationWrapper;
+import com.github.sylvainlaurent.maven.swaggervalidator.semantic.validator.error.SemanticError;
+
+public class MimeTypeValidator extends PathValidatorTemplate {
+
+    @Override
+    public void validate(OperationWrapper operation) {
+        validateMimeTypes(operation);
+        validateConsumes(operation);
+        validateProduces(operation);
+    }
+
+    private void validateConsumes(OperationWrapper operation) {
+        String operationName = operation.getName();
+        List<String> consumes = operation.getConsumes();
+        if (consumes == null) {
+            consumes = new ArrayList<>(emptyIfNull(context.getSwagger().getConsumes()));
+        }
+        if ("get".equals(operationName) || "delete".equals(operationName) ||
```

Comment:
IMO will be better to move "get" and "delete" literals to constants. Please check whether there are already exists such constants in other classes

Reply:
done